azkaban-common/src/main/java/azkaban/executor/container/ContainerFlowCriteria.java

```diff
@@ -0,0 +1,120 @@
+package azkaban.executor.container;
+
+import azkaban.Constants;
+import azkaban.DispatchMethod;
+import azkaban.executor.ExecutableFlow;
+import azkaban.utils.Props;
+import com.google.common.annotations.VisibleForTesting;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.slf4j.Logger;
+
+
+/**
+ * Class for determining {@link azkaban.DispatchMethod} based on flow name filter.
+ */
+public class ContainerFlowCriteria {
+  // Flows are stored in a map where key is project name and value is set of flows.
+  private Map<String, Set<String>> flows = new HashMap<>(1);
+  private final String fileLocation;
+  private final org.slf4j.Logger logger;
```

Comment:
Consider replacing "org.slf4j.Logger logger" to "Logger logger"

Reply:
Ack